pkgs/tools/networking/networkmanager/default.nix

```diff
@@ -216,5 +216,7 @@ stdenv.mkDerivation rec {
     changelog = "https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/raw/${version}/NEWS";
     maintainers = teams.freedesktop.members ++ (with maintainers; [ domenkozar obadz ]);
     platforms = platforms.linux;
+    # Mandatory shared libraries.
+    meta.badPlatforms = [ lib.systems.inspect.patterns.isStatic ];
```

Comment:
Actually, maybe move the comment inside so it is clearer what it belongs to if more bad platforms are added?